routes/config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+// Config
+$router->group([
+    'prefix' => 'config/',
+    'middleware' => ['scope:config']
+], function () use ($router) {
+    // Public access
+    $router->get('/', ['uses' => 'ConfigController@index']);
+    // @todo stop using this in client, and remove?
+    $router->options('/', ['uses' => 'ConfigController@indexOptions']);
+    $router->get('/{id}', ['uses' => 'ConfigController@show']);
+
+    // Restricted access
+    $router->group(['middleware' => ['auth:api', 'scope:config']], function () use ($router) {
```

Comment:
Not sure scope is needed here since the outer route has it, I thought they inherrited.